horizontal-profile/FISViewController.m

```diff
@@ -0,0 +1,195 @@
+    {
+        if (newCollection.verticalSizeClass == UIUserInterfaceSizeClassRegular) // PORTRAIT
+        {
+            [self.view removeConstraints: self.textView6PlusLandscape];
+            [self.view addConstraints: self.textView6PlusPortrait];
+        }
+        else if (newCollection.verticalSizeClass == UIUserInterfaceSizeClassCompact) // LANDSCAPE
+        {
+            [self.view removeConstraints: self.textView6PlusPortrait];
+            [self.view addConstraints: self.textView6PlusLandscape];
+        }
+    }
+    else
+    {
+        if (newCollection.verticalSizeClass == UIUserInterfaceSizeClassRegular) // PORTRAIT
+        {
+            [self.view removeConstraint: self.coverImageHeightLandscape];
+            [self.view addConstraint: self.coverImageHeightPortrait];
+        }
+        else if (newCollection.verticalSizeClass == UIUserInterfaceSizeClassCompact) // LANDSCAPE
+        {
+            [self.view removeConstraint: self.coverImageHeightPortrait];
+            [self.view addConstraint: self.coverImageHeightLandscape];
+        }
+    }
+    NSLog(@"transitioning to: %@", newCollection);
+    
+    [self.view layoutSubviews];
+}
+
+
+-(void) setCoverImageConstraints
+{
+    NSLayoutConstraint *pinToTop = [NSLayoutConstraint constraintWithItem:self.coverImage attribute:NSLayoutAttributeTop relatedBy:NSLayoutRelationEqual toItem:self.view attribute:NSLayoutAttributeTop multiplier:1.0 constant:0];
+    
+    NSLayoutConstraint *setWidth = [NSLayoutConstraint constraintWithItem:self.coverImage attribute:NSLayoutAttributeWidth relatedBy:NSLayoutRelationEqual toItem:self.view attribute:NSLayoutAttributeWidth multiplier:1.0 constant:0];
+    
+    NSLayoutConstraint *setHeight = [NSLayoutConstraint constraintWithItem:self.coverImage attribute:NSLayoutAttributeHeight relatedBy:NSLayoutRelationEqual toItem:self.view attribute:NSLayoutAttributeHeight multiplier:.25 constant:0];
+    
+    NSLayoutConstraint *setCenter = [NSLayoutConstraint constraintWithItem:self.coverImage attribute:NSLayoutAttributeCenterX relatedBy:NSLayoutRelationEqual toItem:self.view attribute:NSLayoutAttributeCenterX multiplier:1 constant:0];
+    
+    // save these guys for later 
+
+    NSLayoutConstraint *setCIHeightLandscape = [NSLayoutConstraint constraintWithItem:self.coverImage attribute:NSLayoutAttributeHeight relatedBy:NSLayoutRelationEqual toItem:self.view attribute:NSLayoutAttributeHeight multiplier:0 constant:0];
+    self.coverImageHeightLandscape = setCIHeightLandscape;
+    self.coverImageHeightPortrait = setHeight;
+    
+    [self.view addConstraints:@[pinToTop, setWidth, setHeight, setCenter]];
+}
+
+-(void) setProfileImageConstraints
+{
+    NSLayoutConstraint *fromCoverImage = [NSLayoutConstraint constraintWithItem: self.profileImage attribute:NSLayoutAttributeTop relatedBy:NSLayoutRelationEqual toItem: self.coverImage attribute:NSLayoutAttributeBottom multiplier:1.0 constant:20];
```

Comment:
those `20` guys? Move them into a constant at the top called like `PADDING` or something